deploy_nva_api.yml

```diff
@@ -25,6 +25,9 @@ Parameters:
     Type: String
     Description: Address to Alma-SRU
     Default: '{{resolve:secretsmanager:AlmaSruHost:SecretString}}'
+  PublicationsTableStreamArn:
+    Type: String
+    Description: ARN of DynamoDB stream events for publications
```

Comment:
Vet vi om dette er beste måtte å koble in en ekstern ressurs? Lurer på om vi skal forske litt på mer dynamiske løsninger i framtiden.